### PR TITLE
feat(github): add pull request template

### PR DESCRIPTION
**Pull request by hairmare**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (eyenx) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]:
```suggestion
* [ ] I added the changes to the docs and updated them using  `pre-commit run -a`
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (eyenx) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:15]: How can we know if CI is green before opening the PR?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (eyenx) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: Shouldnt the changes be tested before applying them?

**Review by eyenx — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:15]: By looking at the results of github actions on your fork (the tests run on each push).

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: They should be tested through GitHub Actions before merging but you also need to test rolling out the changes after the merge (aka don't update things like kps w/o being ready to roll out the change to at least one (dev) env).

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]: We have not been adding changes to the docs so far (as in a changelog per chart) so I'd like to use a different wording here.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:15]: You can also create a PR as "Draft" and then the test results will be easy to find but the PR will not get auto-assigned to a CODEOWNER until you mark it as ready.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (eyenx) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]: okay. But is the phrase "I added to the docs" correct? seems wrong to me. But english is not my native lang.

Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]: i don't like current phrasing either, i'll try to come up with a better one

Comment (tongpu) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]:
```suggestion
* [ ] I updated the charts README.md using `pre-commit run -a`
```
The structure of the comments in `values.yaml` should probably be documented in `CONTRIBUTING.md`.

**Review by tongpu — COMMENTED** [review]
What do you think about this wording?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]: the "adding to the docs" part tried to allude to the existence of the `docs/` folder and the fact that we should not forget to document high-level concepts in there, i might split this into two bullets.

**Review by eyenx — APPROVED** [review]
Better

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (tongpu) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:26]:
```suggestion
* [ ] I documented any high-level concepts I introduced in `docs/`
```

**Review by tongpu — COMMENTED** [review]
One last typo, but other then that: LGTM!

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (hairmare) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: it was intended to be in present tense and I now changed it to "concepts I'm introducing"

Comment (tongpu) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: But how can you use present tense when the sentence starts in past tense? But the version with "concepts I'm introducing" sounds good.

**Review by tongpu — APPROVED** [review]
:shipit:

**Pull request merged**
